### PR TITLE
feat: adds German UI translations

### PR DESCRIPTION
**Pull request by trueberryless**
add section in translations.ts for German translation

### CODE REVIEW

**Comment by trueberryless** [comment]
Wait, how cool is this automation of the documentation?!?

image

This guy is INSANELY smart again, love it!

File: packages/starlight-blog/translations.ts [hunk 1]
```diff
@@ -37,4 +37,23 @@ export const Translations = {
     'starlightBlog.tags.count_one': "{{count}} article avec l'étiquette « {{tag}} »",
     'starlightBlog.tags.count_other': "{{count}} articles avec l'étiquette « {{tag}} »",
   },
+  de: {
+    'starlightBlog.authors.count_one': '{{count}} Beitrag von {{author}}',
+    'starlightBlog.authors.count_other': '{{count}} Beiträge von {{author}}',
+    'starlightBlog.pagination.prev': 'Vorheriger Beitrag',
+    'starlightBlog.pagination.next': 'Nächster Beitrag',
```
Comment (Ergberg) [inline packages/starlight-blog/translations.ts:44]:
```suggestion
    'starlightBlog.pagination.prev': 'Neuere Beiträge',
    'starlightBlog.pagination.next': 'Ältere Beiträge',
```

**Review by Ergberg — COMMENTED** [review]
Hi @trueberryless,

das ist ja toll, danke für die Übersetzung!

even if they are called "prev" and "next", I guess "newer" and "older" better fits the meaning chosen in the other languages.

For "featured", I wondered if "empfohlener Beitrag" would sound more natural, but have no strong opinion (and in deed it is not "recommended").

**Comment by trueberryless** [comment]
> For "featured", I wondered if "empfohlener Beitrag" would sound more natural, but have no strong opinion (and in deed it is not "recommended").  

Yeah, initially I thought this too, then I changed it because this exact reason, but now that you confirmed that it sounds more natural, I changed it back to "empfohlen"...

Thanks for the tip with prev and next, I confused them with the prev and next buttons on single blog post sites... my bad

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks you both for the translations and the reviews :star2:

**Pull request merged**